test/canary/canary_test.go

```diff
@@ -22,6 +24,9 @@ import (
 
 const (
 	configInputPath = "resources/canary_config.json"
+	// todo: expand testing to include regional S3 links, SSM, ECR.
+	downloadLink = "https://amazoncloudwatch-agent.s3.amazonaws.com/amazon_linux/amd64/latest/amazon-cloudwatch-agent.rpm"
```

Comment:
will this work on non-amd64 hosts?

Reply:
definitely not. Good catch. I think the terraform is only launching AL2 x86-64 at the moment.
A future task can take care of testing the windows, ARM, DEB, etc.

Reply:
from the other PR:
```
          command: |
            cd terraform/ec2/linux
            terraform init
            if terraform apply --auto-approve \
              -var="github_test_repo=${{env.CWA_GITHUB_TEST_REPO_URL}}" \
              -var="github_test_repo_branch=${{env.CWA_GITHUB_TEST_REPO_BRANCH}}" \
              -var="user=ec2-user" \
              -var="ami=cloudwatch-agent-integration-test-al2*" \
              -var="arc=amd64" \
```
terraform has amd64 in it